sql/catalyst/src/main/antlr4/org/apache/spark/sql/catalyst/parser/SqlBase.g4

```diff
@@ -375,8 +363,18 @@ ctes
     : WITH namedQuery (',' namedQuery)*
     ;
 
+informationQuery
+    : SHOW (DATABASES | NAMESPACES) ((FROM | IN) multipartIdentifier)? (LIKE? pattern=STRING)?              #showNamespaces
+    | SHOW TABLES ((FROM | IN) multipartIdentifier)? (LIKE? pattern=STRING)?                                #showTables
```

Comment:
Why do not support `SHOW TABLE EXTENDED`?

Reply:
OK